kitsune/users/tests/test_tasks.py

```diff
@@ -30,8 +30,8 @@ def test_process_delete_user(self):
         # Populate inboxes and outboxes with messages between the user and other users.
         other_users = UserFactory.create_batch(2)
         for sender in other_users:
-            send_message([user], "foo", sender=sender)
-        send_message(other_users, "bar", sender=user)
+            send_message([user], to_group="", text="foo", sender=sender)
```

Comment:
In `users/tests/test_views.py` you are using `to_group=None`. Let's be consistent and use either `""` or `None`